tests/test_integration_all_rust.py

```diff
@@ -915,24 +918,35 @@ def test_empty_message_with_crypto_headers(self):
 
         yield self.shut_down(client)
 
+    """
+    # Need to dig into this test a bit more. I'm not sure it's structured correctly
```

Comment:
_maybe_ double check that you have the latest autopush running in this venv? The test Client change for this is [here](https://github.com/mozilla-services/autopush/commit/80fba108123c1e2bfd7a0d71a54ee45cc35e182d#diff-12d6efa75b2f9952a4fcf545d114b9f7)